### PR TITLE
The worker/2 function from the Supervisor.Spec module is deprecated

### PR DESCRIPTION
**Pull request by 0dayfall**
Instead of using Supervisor.Spec.worker/2, you should define your child processes within the Supervisor module using the child_spec/1 function.

### CODE REVIEW

**Review by iamvery — APPROVED** [review]
Thanks!

**Comment by iamvery** [comment]
It looks like we may have to update the Elixir used for the build in order to get this passing.

**Review by iamvery — CHANGES_REQUESTED** [review]
It looks like we may have to update the Elixir used for the build in order to get this passing.

**Comment by 0dayfall** [comment]
> It looks like we may have to update the Elixir used for the build in order to get this passing.

Ok, updated to Elixir v1.5

File: .github/workflows/elixir.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
     - name: Set up Elixir
       uses: erlef/setup-beam@988e02bfe678367a02564f65ca2e37726dc0268f
       with:
-        elixir-version: '1.12.3' # Define the elixir version [required]
+        elixir-version: '1.5' # Define the elixir version [required]
```
Comment (iamvery) [inline .github/workflows/elixir.yml:24]: ❓ What this meant to be 1.15? I think the latest is 1.16.

Comment (0dayfall) [reply .github/workflows/elixir.yml:24]: Ok, I should change that

Comment (iamvery) [reply .github/workflows/elixir.yml:24]: See #288

**Pull request closed without merging**